compute/instances/create-start-instance/create_instance_test.go

```diff
@@ -199,3 +199,100 @@ func TestComputeCreateInstanceSnippets(t *testing.T) {
 		t.Errorf("deleteDisk got err: %v", err)
 	}
 }
+
+func TestComputeBulkCreateInstanceSnippets(t *testing.T) {
+	//ctx := context.Background()
+	//var r *rand.Rand = rand.New(
+	//	rand.NewSource(time.Now().UnixNano()))
+	//tc := testutil.SystemTest(t)
+	//zone := "europe-central2-b"
+	//instanceName := fmt.Sprintf("test-instance-%v-%v", time.Now().Format("01-02-2006"), r.Int())
+	//bootDiskName := fmt.Sprintf("test-disk-%v-%v", time.Now().Format("01-02-2006"), r.Int())
+	//diskName2 := fmt.Sprintf("test-disk-%v-%v", time.Now().Format("01-02-2006"), r.Int())
+	//networkName := "global/networks/default"
+	//subnetworkName := "regions/europe-central2/subnetworks/default"
+	//expectedResult := "Instance created"
+
+	/////
+	ctx := context.Background()
+
+	instanceTemplatesClient, err := compute.NewInstanceTemplatesRESTClient(ctx)
+	if err != nil {
+		t.Fatalf("NewInstanceTemplatesRESTClient: %v", err)
+	}
+	defer instanceTemplatesClient.Close()
+
+	var seededRand *rand.Rand = rand.New(
+		rand.NewSource(time.Now().UnixNano()))
+	tc := testutil.SystemTest(t)
+	zone := "europe-central2-b"
+	instanceTemplateName := "test-instance-template" + fmt.Sprint(seededRand.Int())
+	machineType := "n1-standard-1"
+	sourceImage := "projects/debian-cloud/global/images/family/debian-10"
```

Comment:
```suggestion
	sourceImage := "projects/debian-cloud/global/images/family/debian-12"
```
debian-10 is near its end of life. using a newer release will buy us more time before we need to update these tests.

Reply:
👍